models/uniswap/celo/uniswap_celo_schema.yml

```diff
@@ -0,0 +1,192 @@
+version: 2
+
+models:
+  - name: uniswap_v3_celo_trades
+    meta:
+      blockchain: celo
+      sector: dex
+      project: uniswap_v3
+      contributors: jeff-dude, masquot, soispoke, tomfutago
+    config:
+      tags: ['celo','uniswap_v3','trades', 'uniswap','dex']
+    description: >
+        Uniswap V3 contract trades on Celo
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - block_date
+            - blockchain
+            - project
+            - version
+            - tx_hash
+            - evt_index
+      - check_dex_seed:
+          blockchain: celo
+          project: uniswap
+          version: 3
```

Comment:
In order for this test to work, we can add some seed records in this file:
https://github.com/duneanalytics/spellbook/blob/main/seeds/dex/trades/dex_trades_seed.csv